src/annotator/plugin/test/pdf-metadata-test.js

```diff
@@ -14,7 +14,7 @@ describe('pdf-metadata', function () {
     window.dispatchEvent(event);
 
     return pdfMetadata.getUri().then(function (uri) {
-      assert.equal(uri, 'http://fake.com');
+      assert.equal(uri, 'http://fake.com/');
```

Comment:
The trailing `/` is being added here because `getUri()` now returns a resolved URL so if `PDFViewerApplication.url` is `http://example.com`, the result will be `http://example.com/`.